css/wishlist.css

```diff
@@ -0,0 +1,38 @@
+.wishlist-card-container {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  row-gap: 2rem;
+}
+.wishlist-card {
+  max-width: 320px;
+  border: 1px solid var(--light-gray);
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+  padding-top: 0.75rem;
+  position: relative;
+  row-gap: 1rem;
+}
+.wishlist-card-img-container {
+  text-align: center;
+}
+.wishlist-card-img {
+  width: 50%;
+}
+.wishlist-card-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: stretch;
+  row-gap: 0.5rem;
+}
+.wishlist-btn {
+  align-self: stretch;
+  border-radius: 0;
```

Comment:
Default value of border radius is 0 , you don't need to define that.

Reply:
It has two classes one come from the component library which has a border-radius so that why I have added 0 as I don't need it